core/domain/exp_domain.py

```diff
@@ -2460,6 +2513,48 @@ def get_all_html_content_strings(self):
 
         return html_list
 
+    @classmethod
+    def to_proto(
+            cls, exploration_id, title, version, init_state_name, states):
+        """Calculate the exploration size by setting exploration proto.
+
+        Args:
+            exploration_id: str. The id of the exploration.
+            title: str. The exploration title.
+            version: int. The version of the exploration.
+            init_state_name: str|None. The name of the initial state.
+            states: list|None. The list of state
+                domain objects.
+
+        Returns:
+            Proto Object. The exploration proto object.
```

Comment:
What do you get when you do `type(exploration_proto)`? Typically, that is what gets written as the "type" in the docstring.

Reply:
Type = `<class 'exploration_pb2.Exploration'>`